2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,18 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let obj = {};
+countryCurrencyCodes.forEach((elem) => {
+    obj[`${elem[0]}`] = elem[1]
+});
+
+return obj;
+}
+
+let result = createLookup(COUNTRY_CURRENCY_CODES);
+
+
+console.log(result);
 }
```

Comment:
Something tells me you haven't run this code, or the test. It's got the wrong number of `}` or `{`